### PR TITLE
Removes shotguns from maps, being made and ordered

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Simple pr
The bartender keeps their shotgun but can only use the rubber shot they get roundstart for it
Replaced with a few extra eguns in armouries

## Why It's Good For The Game

Shotguns bad for balance and only used for powergame

been a while coming

image

## Changelog
:cl:
del: Shotgun ammo and shotguns can no longer be crafted
balance: No more shotguns in armouries or cargo
/:cl:

### CODE REVIEW

**Comment by Timberpoes** [comment]
If this is a serious decree from 🍊 then who are we as mortals to judge Him for His divine plan?

**Comment by CRITAWAKETS** [comment]
you could also just rebalance shotguns, making them do 40 damage instead of 80 is somehow actually sensible

i guess this is what i get for taking too long to do it

**Comment by Fikou** [comment]
le funny bad faith pr

**Comment by CoffeeDragon16** [comment]
I don't see what's to gain with this PR. Shotguns diversify ranged combat by providing a strong counter to melee stun combat. They're also simply fun.

**Comment by wesoda25** [comment]
There are numerous ways to balance shotguns. Make them all only fit on backslot, reduce how many are available, and just nerf their damage, to name a few. Why aren’t we trying these things before flat out removal?

**Comment by TJatPBnJ** [comment]
removes the techshell crafting recipes but not the research or empty techshells
this is a good pr

**Comment by 81Denton** [comment]
>Shotguns bad for balance
How?

**Comment by zxaber** [comment]
The buff (s)laughter demons really needed.

**Comment by Capsandi** [comment]
Good PR, the real reason shotties need to go is cause they ain't energy weapons.

**Comment by Ryll-Ryll** [comment]
After the work I've done tuning the pellet cloud component and its interactions with wounds/the mechanics of piercing wounds causing dismemberment, I'd be extremely sad to see this go through, but it's in the maintainers' hands now

**Comment by saprasam** [comment]
i hate coders

**Comment by DinkleNH** [comment]
"Shotguns bad for balance and only used for powergame" Elaborate. By far the worst worded PR I have ever seen. You cant just say "it powergame" and "Bad for balance", you have to EXPLAIN why it is X and Y, and replacing them with eguns? How does that fix balance?

**Comment by githubuser4141** [comment]
I mean lore-wise it makes sense that NT would make ballistics hard to obtain seeing as a good portion of the station's exterior walls are made out of fragile glass and relatively thin metal paneling. And we'd still have mech ballistics so it's not that bad.

**Comment by SkatesMcGates** [comment]
I have to disagree with this PR. To echo some previous sentiments, shotguns have their cons (limited amount of ammunition, need to rack riot shotguns, they take up a lot of space, etc.). And that's not to say they shouldn't be nerfed at all, I could get behind a different way to make them less powerful, sure (especially Cargo's combat shotguns). But to completely remove them carries a lot of issues.

One potentially unforeseen aspect of this PR is the huge buff this gives to a lot of antags. The Riot Shotguns are some of the station's best defenses to a wizard, nuke ops, slaughter demons, nightmares, etc. which are some of the most powerful antagonists there are. There's also the simple but important point that shotguns are enjoyable and fun to use.

Additionally, this PR is unfortunately lacking in detail or explanation. This is a large change to the game and it would be valuable to see both points of view fully fleshed out would help make the decision more clear. I would say these are not obviously overpowered, I haven't seen anyone immediately rushing the armory for shotguns or seeing shotguns used every single round, in fact (at least in my experience) the vast majority of the time the shotguns are used is to counter nuke ops/wizard/demons.

It seems to me like a lot of changes recently, such as making nearly all lethal laser weapons bulky, have been moving to reduce the amount of lethality in the game. This is a pretty subjective matter but I thought the level of lethality in the game was at a good place. As the rules even state, dying and losing is pretty normal, some weapons are pretty powerful and some kind of suck, sometimes you lose a good fight and sometimes there's not a whole lot you could do to counter, and that's okay. 

In conclusion, boomstick good, no boomstick bad

**Comment by Pugie** [comment]
Lame!

**Comment by MrDoomBringer** [comment]
yes officer i am making this PR
yes office this PR will get merged
no officer there's nothing you can do about it

**Comment by RuneQuester** [comment]
As many have said, this PR describes none of their arguments, and shotguns are fairly balanced. YOu have to get close to them, and many weapons just counter the shotgun

**Comment by theboy6545** [comment]
mmmm orang my fruit of favor

**Comment by Capsandi** [comment]
If your removing the makeshift shotgun, you gotta remove the modular receiver and wooden stock, as their only usage is to make the makeshift shotgun

**Comment by ShibaInuLord** [comment]
i use(d) the shotgun

**Comment by InvalidArgument3** [comment]
I disagree, because they are cool

**Comment by Moja10** [comment]
I'm a powergamer who uses shotguns, and we are the most persecuted race on the planet.

**Comment by actioninja** [comment]
You fell for the orange man bait

**Comment by trexter555** [comment]
Cause the shot gun doesn't miss, and unlike the shitty hybrid taser it stops a criminal in their tracks in two hits. Bang, bang, and they're fucking done. I use four shots just to make damn sure. Because, once again, I'm not there to coddle a buncha criminal scum sucking f------, I'm there to 1) Survive the fucking round. 2) Guard the armory. So you can absolutely get fucked. If I get unbanned, which I won't, you can guarantee I will continue to use the shotgun to apprehend criminals. Because it's quick, clean and effective as fuck. Why in the seven hells would I fuck around with the disabler shots, which take half a clip just to bring someone down, or with the tazer bolts which are slow as balls, impossible to aim and do about next to jack shit, fuck all. The shotgun is the superior law enforcement weapon. Because it stops crime. And it stops crime by reducing the number of criminals roaming the fucking halls.

**Comment by ExcessiveUseOfCobblestone** [comment]
Please give them 24 hours to correct their PR body/desc rather than early-closing

**Comment by MrDoomBringer** [comment]
tiviplus caught being an orange henchmen

**Comment by optimumtact** [comment]
This isn't bait I intend to do this, just gotta find the right time

**Comment by MrDoomBringer** [comment]
@actioninja fell for the orange-man-bait bait

**Comment by optimumtact** [comment]
it has to be considered as part of a larger overhaul of guns so he's not exactly wrong
